### ISSUE TITLE
It doesn't work

### ISSUE THREAD

**Issue by do0rknob** [behnamazimi/in-site-shorteys#7]
I enabled the extension for the website that I wish to add a click shortcut on. Then I click on "Add click shortcut", and then I click on a website element (a button in this case), and then I choose a key combination, and then nothing happens.

EDIT: After explicitly allowing popups in my browser settings and then clicking on "Add click shortcut" while viewing the page that I would like to add a shortcut on I was able to see the popup "dialog" for "Action steps:" with the green background in the lower right corner and here's what happened:

I went directly to the element I want to add a a shortcut for (a link button) and clicked it, and above step 1 in the "Action steps:" it now says "unknown" and shows "step 2" to the right. I then clicked on "Set Keys", chose a key combination, and pressed "Add". Then I pressed the key combination and a different popup "dialog" appeared in the lower right corner that reads "Step calling failed because the element was not found. Maybe you are in wrong page."

I went to delete this shortcut and try to create it again by clicking a different element, the link itself (next to the button), and add it again. When I tried to set the key combination a message popped up that said "Shortcut used before". I went into the settings and clicked to erase all shortcuts, went back again to add the shortcut, and again was told "Shortcut used before".

I also tried to click inside the popup "dialog" and name the shortcut and my clicks inside the action steps popup were recorded as steps in the shortcut I was trying to create. Obviously this is not going to work.

tl;dr It doesn't work

**Comment by behnamazimi** [comment]
Is it like this on all websites?

**Comment by do0rknob** [comment]
That is a great question and before I answer I would like to thank you for
replying to my (first time on GitHub) comment. I can confirm that it was
like this on all websites that I tested it on because I did not test it on
any other websites. Furthermore I now realize that there is some
information I overlooked (and also failed to provide in my original
comment) which might be relevant to why it did not work on the one site I
tested it on. Now that I think about it there are a few reasons why it may
not have worked and all together those would be... I tested it on a non-ssl
site on my LAN served from a non-standard port (8123) using a domain that
resolved via mdns, which was "homeassistant" ...in case the port number
didn't give that away. I do apologize for not testing it on an actual
website, accessible to the public, out in the wild, which was not operating
under a set of somewhat abnormal circumstances. I would be happy to test it
on a website which meets all of the standard criteria when I am able.
Thanks again for acknowledging my comment.
